Sources/Controllers/MessagesViewController.swift

```diff
@@ -45,6 +45,12 @@ open class MessagesViewController: UIViewController {
     ///
     /// The default value of this property is `false`.
     open var scrollsToBottomOnKeybordBeginsEditing: Bool = false
+    
+    /// A Boolean value that determines whether the `MessagesCollectionView` scrolls
+    /// according to the change of the height of the keyboard.
+    ///
+    /// The default value of this property is `false`.
+    open var scrollsOnKeyboardFrameChanges: Bool = false
```

Comment:
Let's change `scrollsOnKeyboardFrameChanges` to `maintainPositionOnKeyboardFrameChange`